test/com/esotericsoftware/kryo/serializers/DefaultSerializersTest.java

```diff
@@ -457,6 +459,27 @@ public void testURLSerializer () throws Exception {
 		a, b, c
 	}
 
+    @Test
+    public void testConcurrentSkipListMapSerializer() {
+        HashMap map = new HashMap();
+        kryo.register(ConcurrentSkipListMap.class);
+		kryo.register(Comparator.reverseOrder().getClass());
+        ConcurrentSkipListMap concurrentSkipListMap = new ConcurrentSkipListMap(Comparator.reverseOrder());
+        map.put(9, "456");
+        map.put(3, "abc");
+        map.put(1, 122);
+        map.put(2, 314);
+        map.put(4, 2511);
+        map.put(8, "hello");
+        map.put(10, "wasted");
+        map.put(5, "Great");
+
+        concurrentSkipListMap.putAll(map);
+        ConcurrentSkipListMap concurrentSkipListMapCopy = kryo.copy(concurrentSkipListMap);
+        assertEquals(concurrentSkipListMap.comparator(), concurrentSkipListMapCopy.comparator());
+        roundTrip(55, concurrentSkipListMap);
```

Comment:
Please also add a test for sub-classes of `ConcurrentSkipListMap`. See `MapSerializerTest.testTreeMap()`.